python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -42,12 +73,13 @@ def get_dimensions(dataset):
     return dimensions, types
 
 
-def get_partitions(dku_flow_variables, dimensions):
+def get_partitions(dku_flow_variables, dimensions, source=None):
```

Comment:
Same comment as previously, I would advise using the terms input/output instead of origin/source/destination which are not used in DSS.